src/Razor/src/rzls/RazorConnectHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
+using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+using Microsoft.CodeAnalysis.Razor.Protocol;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.ProjectSystem;
+
+#pragma warning disable RS0035 // External access to internal symbols outside the restricted namespace(s) is prohibited
```

Comment:
@davidwengier I'm sure this comes from some publicapi/bannedsymbols/other thing. Do you know?

Reply:
If it isn't, it would have to have a conditional MEF import for something that will only be available in VS Code. Having this in rzls to me makes sense to me. I'm not strongly opinionated though